File: Java/Nesneye Day.Prog  (Java)/Nesneye Day.Prog  (Java)/Nesneye Day.Prog  (Java)/Java Uygulamalar/5. Hafta Uygulamaları/Uygulamalar5/src/Uygulama5.java

public class Uygulama5 {

	public static void main(String[] args) {
		int[][]dizi=new int[4][4];
		for (int i = 0; i < dizi.length; i++) {
			for (int j = 0; j < dizi.length; j++) {
				if(i==j){
					dizi[i][j]=0;
				}
				else
				{
					dizi[i][j]=i*4+j+1;
				}}}
		for (int i = 0; i < dizi.length; i++) {
			for (int j = 0; j < dizi.length; j++) {
				System.out.print(dizi[i][j]+"\t");
			}
			System.out.println();
		}}}

File: Oracle/02.12.2015/Yeni klasör (2)/Uygulama1/uygulama5/uygulama5/uygulama5.cpp
// uygulama5.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <iostream>

using namespace std;

void _tmain(int argc, _TCHAR* argv[])
{
	int elemansayisi;
	cout << "Eleman Sayisini giriniz:";
	cin >> elemansayisi;
	double dizi =new double [elemansayisi];
	for (int i = 0; i < elemansayisi; i++){
		dizi[i] = i + 0.1;
	}
}

